subsys/bluetooth/CMakeLists.txt

```diff
@@ -4,7 +4,12 @@ target_include_directories(subsys__bluetooth INTERFACE ${CMAKE_CURRENT_SOURCE_DI
 
 add_subdirectory(common)
 add_subdirectory_ifdef(CONFIG_BT_HCI host)
-add_subdirectory_ifdef(CONFIG_BT_CTLR controller)
 add_subdirectory_ifdef(CONFIG_BT_SHELL shell)
 
+if(CONFIG_BT_CTLR)
```

Comment:
`if(CONFIG_BT_CTLR AND CONFIG_BT_LL_SW)`

Reply:
I prefer to keep it as is, for now it is only one BT_CTLR option, and that is BT_LL_SW, but in the future there might be more, then the inner if has some else if options